docs/documentation/development/bindings/thing-definition.md

```diff
@@ -169,51 +170,6 @@ Some channels might have only a limited and countable set of states. These state
 
 The user interface can use these values to render labels for values or to provide a selection of states, when the channel is writable. The option labels can also be localized.
 
-### Channel Categories
```

Comment:
I just found a link to this section on https://www.eclipse.org/smarthome/documentation/features/icons.html (which is currently already broken) - I guess that needs to be changed as well. Did you check if there are any other references that need to be updated?